thrift/messaging.thrift

```diff
@@ -192,7 +192,7 @@ service BlockchainService {
 
    list<string> receipt_request(1: string signatory),
 
-   list<VerificationRecord> transfer_data(1: list<string> received, 2: list<string> unreceived),
+   list<VerificationRecord> transfer_data(1: string transfer_to, 2: list<string> received, 3: list<string> unreceived),
```

Comment:
Could we figure out the `transfer_to` id based on the pass phrase used when the connection is registered instead of passing in any arbitrary node id?